lms/templates/ccx/grading_policy.html

```diff
@@ -3,7 +3,7 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
+    <div class="msg-content" tabindex="0">
```

Comment:
@pwilkins great. Please be sure to include the test plan instructions in the description of every PR you open, especially what specific scenario(s) to test to validate whatever change is being made. The reasoning for this is different engineers may be reviewing subsequent PRs, also they may just simply forget about this particular feature and need the prodding. Please see [these guidelines about building an excellent PR description](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/cover-letter.html) - it's really helpful if you can provide us the most amount of context and information (test scenarios, screenshots, etc) with every pull request. I promise, it makes the process go faster.